legate/driver/launcher.py

```diff
@@ -83,6 +84,15 @@ def __init__(self, config: ConfigProtocol, system: System) -> None:
         self._config = config
         self._system = system
 
+        if config.multi_node.ranks == 1:
+            self.rank_id = "0"
```

Comment:
@manopapad I guess maybe we could lose this first condition, `bind.sh` should be taking care of this too